artemis/config.py

```diff
@@ -698,6 +698,29 @@ class Nuclei:
                 "NUCLEI_TEMPLATE_CHUNK_SIZE is 200, three calls will be made with 200 templates each.",
             ] = get_config("NUCLEI_TEMPLATE_CHUNK_SIZE", default=200, cast=int)
 
+        class PlaceholderPagesHtmlElements:
+            PLACEHOLDER_PAGE_HTML_ELEMENTS: Annotated[
+                List[str],
+                "A list of html elements that appear on placeholder pages",
```

Comment:
maybe write that exact string matching will be performed, no parsing, no nothing. And write what will happen with such pages